wsd/wopi/StorageConnectionManager.hpp

```diff
@@ -171,7 +78,6 @@ class StorageConnectionManager final
     /// Saves new URI when resource was moved
     // void setUri(const Poco::URI& uri) { _uri = sanitizeUri(uri); }
 
-    static bool FilesystemEnabled;
```

Comment:
I assume this is a system property alike flag? Potentially immutable after setup?
Then we could use a singleton (via a local static instance) of a property class object,
which also takes care of the mutability etc? @Ashod (just brainstorming)

Reply:
This is only meaningful in debug builds, for debugging.